R2API/ItemDrop/DropList.cs

```diff
@@ -25,39 +26,64 @@ public class DropList {
                 Rather than whichever api is executed second using the first api's altered lists as a base.
             */
 
-            private static System.Reflection.FieldInfo availableTier1DropListInfo = typeof(Run).GetField("availableTier1DropList", System.Reflection.BindingFlags.Public | System.Reflection.BindingFlags.Instance);
-            private static System.Reflection.FieldInfo availableTier2DropListInfo = typeof(Run).GetField("availableTier2DropList", System.Reflection.BindingFlags.Public | System.Reflection.BindingFlags.Instance);
-            private static System.Reflection.FieldInfo availableTier3DropListInfo = typeof(Run).GetField("availableTier3DropList", System.Reflection.BindingFlags.Public | System.Reflection.BindingFlags.Instance);
-            private static System.Reflection.FieldInfo availableEquipmentDropListInfo = typeof(Run).GetField("availableEquipmentDropList", System.Reflection.BindingFlags.Public | System.Reflection.BindingFlags.Instance);
-
             public static bool OriginalListsSaved;
+            public static List<ItemIndex> AvailableItemsOriginal;
+            public static List<EquipmentIndex> AvailableEquipmentOriginal;
+            /* Single-tier item drop lists */
             public static List<PickupIndex> Tier1DropListOriginal = new List<PickupIndex>();
             public static List<PickupIndex> Tier2DropListOriginal = new List<PickupIndex>();
             public static List<PickupIndex> Tier3DropListOriginal = new List<PickupIndex>();
-            public static List<PickupIndex> BossDropListOriginal = new List<PickupIndex>();
-            public static List<PickupIndex> LunarDropListOriginal = new List<PickupIndex>();
+            public static List<PickupIndex> BossDropListOriginal = new List<PickupIndex>();          // RoR2 only puts items, not equipment, in the availableBossDropList
+            public static List<PickupIndex> LunarItemDropListOriginal = new List<PickupIndex>();
+            public static List<PickupIndex> VoidTier1DropListOriginal = new List<PickupIndex>();
+            public static List<PickupIndex> VoidTier2DropListOriginal = new List<PickupIndex>();
+            public static List<PickupIndex> VoidTier3DropListOriginal = new List<PickupIndex>();
+            public static List<PickupIndex> VoidBossDropListOriginal = new List<PickupIndex>();
+            /* Single-tier equipment drop lists */
             public static List<PickupIndex> EquipmentDropListOriginal = new List<PickupIndex>();
-            public static List<PickupIndex> NormalEquipmentDropListOriginal = new List<PickupIndex>();
```

Comment:
Should probably leave around a n [Obsolete] NormalEquipmentDropListOriginal for backwards compatibility.